packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -11579,7 +11579,7 @@ export default function MovieUpdateForm(props) {
             }
           });
           tagsToLinkMap.forEach((count, id) => {
-            const tagToLink = tagRecords.find((r) =>
+            const tagToLink = tagsRecords.find((r) =>
```

Comment:
Here's a bug fix example. Notice how there's no corresponding diff for this variable name above this line. That's because tagRecords should have always been tagsRecords.